include/oneapi/tbb/task_arena.h

```diff
@@ -404,6 +500,33 @@ class task_arena : public task_arena_base {
         return execute_impl<decltype(f())>(f);
     }
 
+#if __TBB_PREVIEW_PARALLEL_PHASE
+    void start_parallel_phase() {
+        initialize();
+        r1::register_parallel_phase(this, 0);
+        // Trigger worker threads to join arena
+        enqueue([]{});
+    }
+    void end_parallel_phase(bool with_fast_leave = false) {
+        __TBB_ASSERT(my_initialization_state.load(std::memory_order_relaxed) == do_once_state::initialized, nullptr);
+        r1::unregister_parallel_phase(this, with_fast_leave);
+    }
+
+    class scoped_parallel_phase {
+        task_arena& arena;
+        bool one_time_fast_leave;
+    public:
+        scoped_parallel_phase(task_arena& ta, bool with_fast_leave = true) 
+            : arena(ta), one_time_fast_leave(with_fast_leave)
```

Comment:
Shouldn't `with_fast_leave` default to `false` here?

Reply:
Done.